fabtests/common/shared.c

```diff
@@ -2404,6 +2406,42 @@ void eq_readerr(struct fid_eq *eq, const char *eq_str)
 	}
 }
 
+int ft_sync_manual_progress()
+{
+	int ret, value = 0, result = 0;
+
+	if (opts.dst_addr) {
+		ret = send(oob_sock, &value, sizeof(value), 0);
+		if (ret != sizeof(value))
+			return -FI_EOTHER;
+
+		do {
+			ret = recv(oob_sock, &result, sizeof(result), MSG_DONTWAIT);
+			if (ret == sizeof(result))
+				break;
+
+			ret = ft_spin_for_comp(rxcq, &rx_cq_cntr, 0, 0, 0);
```

Comment:
Hmm... there is a chance that we can get a completion here for the next iteration of the test.  We need to structure the tests so this works.

Reply:
If we call fi_cq_read(cq, NULL, 0), that will drive progress without actually retrieving a completion.  Problem solved, I think, maybe.